### ISSUE TITLE
__apt_get_upgrade_noinput arguments mistake?

### ISSUE THREAD

**Issue by cvrebert** [saltstack/salt-bootstrap#355]
```
In bootstrap-salt.sh line 1055:
__apt_get_upgrade_noinput() {
^-- SC2120: __apt_get_upgrade_noinput references arguments, but none are ever passed.

In bootstrap-salt.sh line 1517:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1550:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1708:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1785:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1851:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1893:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.

In bootstrap-salt.sh line 1923:
        __apt_get_upgrade_noinput || return 1
        ^-- SC2119: Use __apt_get_upgrade_noinput "$@" if function's $1 should mean script's $1.
```

**Comment by s0undt3ch** [comment]
Where are you getting that "lint" from?

**Comment by cvrebert** [comment]
Like I said in the other issue: https://github.com/koalaman/shellcheck

**Comment by s0undt3ch** [comment]
Sorry, I haven't gone through them all ;)

**Comment by s0undt3ch** [comment]
I will definitely need to dig into that!!!

**Comment by s0undt3ch** [comment]
Fixed in #358
